storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -186,6 +186,8 @@ public Optional<String> serverConfigName(String configName) {
     public static final long DEFAULT_MESSAGE_TIMESTAMP_DIFFERENCE_MAX_MS = ServerLogConfigs.LOG_MESSAGE_TIMESTAMP_DIFFERENCE_MAX_MS_DEFAULT;
 
     public static final boolean DEFAULT_REMOTE_STORAGE_ENABLE = false;
+    public static final boolean DEFAULT_REMOTE_LOG_COPY_DISABLED_CONFIG = false;
```

Comment:
nit:

rename `DEFAULT_REMOTE_LOG_COPY_DISABLED_CONFIG` to `DEFAULT_REMOTE_LOG_COPY_DISABLE_CONFIG`